ui/webui/webui-desktop

```diff
@@ -88,7 +88,7 @@ BROWSER_HOME=$(mktemp --directory --tmpdir cockpit.desktop.XXXXXX)
 # forward parent stdin and stdout (from bridge) to cockpit-ws
 # it pretty well does not matter which port we use in our own namespace, so use standard http
 # disable /etc/cockpit/
-XDG_CONFIG_DIRS="$BROWSER_HOME" COCKPIT_SUPERUSER="pkexec" /usr/libexec/cockpit-ws -p 80 -a 127.0.0.90 --local-session=cockpit-bridge &
+XDG_CONFIG_DIRS="$BROWSER_HOME" COCKPIT_SUPERUSER="pkexec" /usr/libexec/cockpit-ws -p 80 --no-tls --local-session=cockpit-bridge &
```

Comment:
@martinpitt this change here makes the cockpit-ws listen on all possible addresses right? 

Since webui-desktop is not runnign in network namespace, this is probably problematic in matters of security. I tried to put this back, but then port forwarding does not seem to work for 80 port for passing requests from the host. Any ideas?

Reply:
Ah yes, that's a good point. So you either need to support an `${ADDRESS:-127.0.0.1}` variable which you can change from the test, or the test needs to run socat or similar to forward the port. I think the former is easier.